internal/accounts/create.go

```diff
@@ -134,3 +147,301 @@ func create(
+	if err != nil {
+		return nil, err
+	}
+
+	var address flow.Address
+
+	if selectedNetwork == config.DefaultEmulatorNetwork() {
+		log.Info(fmt.Sprintf("%s Creating the account on %s with generated keys", output.WarningEmoji(),
+			networkName))
+		log.StartProgress("")
+		time.Sleep(time.Second * 3)
+
+		signer, err := state.EmulatorServiceAccount()
+		if err != nil {
+			return nil, err
+		}
+		account, err := service.Accounts.Create(
+			signer,
+			[]crypto.PublicKey{key.PublicKey()},
+			[]int{flow.AccountKeyWeightThreshold},
+			[]crypto.SignatureAlgorithm{crypto.ECDSA_P256},
+			[]crypto.HashAlgorithm{crypto.SHA3_256},
+			nil,
+		)
+		if err != nil {
+			return nil, err
+		}
+		log.StopProgress()
+
+		log.Info("\nPlease note, that the newly created account will only be available until you keep the emulator service up and running, if you restart the emulator service all accounts will be reset. If you want to persist accounts between restarts you must use the '--persist' flag when starting the flow emulator.")
+
+		address = account.Address
+	} else {
+		var link string
+		switch selectedNetwork {
+		case config.DefaultTestnetNetwork():
+			log.Info(fmt.Sprintf("%s We will be creating the account on testnet faucet website (%s), which has been prefilled with generated keys", output.WarningEmoji(), util.TestnetFaucetHost))
+			log.Info("\nPlease follow the steps: \n 1. Fill in the captcha, \n 2. Click on 'Create Account' button.\n")
+			output.NextStepPrompt()
+			link = util.TestnetFaucetURL(key.PublicKey().String(), crypto.ECDSA_P256)
+		case config.DefaultMainnetNetwork():
+			log.Info(fmt.Sprintf("%s We will be creating the account on Flow Port website (%s), which has been prefilled with generated keys", output.WarningEmoji(), util.FlowPortUrl))
+			log.Info("\nPlease follow the steps: \n 1. Click on 'Submit' button, \n 2. Connect existing Blocto or create a new account first, \n 3. Click on confirm, \n 4. Click on approve. \n")
+			output.NextStepPrompt()
+			link = util.MainnetFlowPortURL(key.PublicKey().String())
+		}
+		log.StartProgress("Waiting for an account to be created, please finish all the steps in the browser...\n")
+		time.Sleep(time.Second * 2)
+		err := util.OpenBrowserWindow(link)
+		if err != nil {
+			return nil, err
+		}
+
+		addr, err := getAccountCreatedAddressWithPubKey(service, key.PublicKey(), startHeight)
+		if err != nil {
+			return nil, err
+		}
+		address = *addr
+
+		log.StopProgress()
+	}
+
+	onChainAccount, err := service.Accounts.Get(address)
+	if err != nil {
+		return nil, err
+	}
+
+	account, err := flowkit.NewAccountFromOnChainAccount(name, onChainAccount, key)
+	if err != nil {
+		return nil, err
+	}
+	log.Info(fmt.Sprintf("\n %s Successfully created account on %s", output.OkEmoji(), networkName))
+	output.NextStepPrompt()
+
+	log.Info(fmt.Sprintf("\n %s Saving created account in flow.json", output.WarningEmoji()))
+	log.StartProgress("")
+	time.Sleep(time.Second * 3)
+	log.StopProgress()
```

Comment:
I don't think this delay is better, I think good UX should always be as quick as possible and if these steps don't require any interactions there is no need to slow them down.

Reply:
addressed